easybuild/tools/testing.py

```diff
@@ -295,7 +295,7 @@ def post_pr_test_report(pr_nr, repo_type, test_report, msg, init_session_state,
 
     if build_option('include_easyblocks_from_pr'):
         if repo_type == GITHUB_EASYCONFIGS_REPO:
-            easyblocks_pr_nrs = [int(eb_pr_nr) for eb_pr_nr in build_option('include_easyblocks_from_pr')]
+            easyblocks_pr_nrs = [int(x) for x in build_option('include_easyblocks_from_pr')]
```

Comment:
there are a few other very similar list comprehensions for pr numbers, is it worth also changing them? (given https://github.com/easybuilders/easybuild-framework/pull/3724#issuecomment-855379319 and https://github.com/easybuilders/easybuild-framework/pull/3725)

Reply:
Probably worth also using single-letter variables there, yes...

Reply:
fixed in 14715cb